src/access-broker.c

```diff
@@ -589,7 +589,8 @@ access_broker_get_trans_object_count (AccessBroker *broker,
                                  NULL);
     if (rc != TSS2_RC_SUCCESS)
         goto out;
-    *count = capability_data.data.handles.count;
+    if (count != NULL)
```

Comment:
There is a check for it above in line 575 g_assert_nonnull (count);

Reply:
Yes.  The compiler doesn't seem to know that it checks for NULL and (I suppose?) stops execution or something.

Reply:
Any chance we can use something like the coverity model files for this? I had to create a simple model file to teach coverity how g_error works here: https://github.com/tpm2-software/tpm2-abrmd/blob/master/coverity/coverity-model.c

Reply:
That would be nice. I seen it in other static analysis software, but not in scan-build.

Reply:
I thought about this some more and I think I can kludge in a coverity-like model using #define for scan-build.

Reply:
I just punched 'scan-build glib' into the googles and this was the first hit: https://www.freedesktop.org/software/tartan/ ... might be useful.

Reply:
whoops, this isn't the scan-build thread. My mistake. So this is a newer version of GCC presumably with some static analysis stuff built in.